### PR TITLE
Added support for domain-based logins

### PR DESCRIPTION
**Pull request by luiswebmercado**
I really loved this API and I'm using it in a project here in my company that requires this type of configurations...

I does not impact in "non-domain-based" logins.

I added some code into Wikimate's login method and change the README file, adding info about how to inform and where to inform the domain name.

### CODE REVIEW

File: Wikimate.php [hunk 1]
```diff
@@ -62,6 +62,13 @@ public function login( $username, $password ) {
 			'lgpassword' => $password,
 			'format' => 'json'
 		);
+
+		// If $domain is informed, sets the correspondant detail in the request information array
+		if( $domain )
```
Comment (samwilson) [inline Wikimate.php]: Should this be a more robust check, e.g. `$domain !== NULL` or `is_string($domain)`?

**Comment by waldyrious** [comment]
@samwilson, the original author seems to be MIA. I could merge and make the change you propose. Which of your suggestions do you think would be preferable?

File: README.md [hunk 2]
```diff
@@ -32,14 +32,17 @@ to login to that wiki.
...
 $username = 'bot';
 $password = 'password';
-	
+
 try
 {
 	$wiki = new Wikimate($api_url);
+
+    // You can pass the domain name either
+    // $wiki->login($username,$password, $domainName)
```
Comment (waldyrious) [inline README.md]: Indentation needs adjusting.

File: Wikimate.php [hunk 3]
```diff
@@ -62,6 +62,13 @@ public function login( $username, $password ) {
...
+		{
+			$details['lgdomain'] = $domain;
+
```
Comment (waldyrious) [inline Wikimate.php]: Excess whitespace here.

File: Wikimate.php (hunk 1 above)
Comment (samwilson) [inline Wikimate.php]: I reckon `is_string($domain)` would be most readable.

**Comment by luiswebmercado** [comment]
Hey! Sorry the miss attention in this topic, I did not see the notification about these messages.
I agree with everything discussed here.

Thanks for the merge.

**Pull request merged**